net-p2p/bitcoin-core/metadata.xml

```diff
@@ -14,6 +14,7 @@
 		<flag name="bitcoin-cli">Build and install bitcoin-cli command line interface</flag>
 		<flag name="cli">Build and install bitcoin-cli command line interface</flag>
 		<flag name="daemon">Build and install bitcoind daemon</flag>
+		<flag name="extended-tests">Run tests that take a long time</flag>
```

Comment:
See https://public-inbox.gentoo.org/gentoo-dev/87zfplr9g0.fsf@gentoo.org/.

Reply:
In this case, the word "extended" comes from the command-line flag passed to Bitcoin Core's test runner. If you would prefer an alternative `USE` flag, I am fine to change it. Just say what.

Reply:
Let's do `test-full`.

Reply:
Note that running the extended tests was the default in the ebuilds of previous releases, but they really do take much longer (a couple of hours for me) — ­so much longer, in fact, that upstream gates them behind an optional `--extended` flag. I had been specifying that flag unconditionally for maximum test coverage, but I'd like to give users the option to trade less exhaustive test coverage for a quicker merge.

Reply:
> Let's do `test-full`.

Do I need to describe that in `metadata.xml`, or is it intended to be a global `USE` flag?

Reply:
Add a description for now, and later, I'll go over all local descriptions and remove them where they don't provide any useful additional package-specific context. If you can think of some detail to add there that is specific to Bitcoin and worth keeping, then do so and I won't remove it. Otherwise, just make it generic.

Reply:
Done in 0cd179a0bb1dd3753bcb88e03325d3aaf309ffa0.